procedures/accounts_setup.md

```diff
@@ -0,0 +1,250 @@
+Day 1 checklist
+===============
+
+Welcome to Release Engineering!
+This page is meant to get new hires, interns, or interested community members up to speed with the right software,
+configurations, and communication channels to contribute effectively to the release engineering pipeline.
+
+## Basic
+
+The accounts get activated 1 day before joining so the manager of the team should drop a Slack message and an email to ensure comms are working properly.
+Moreover, your manager should have already scheduled a 1x1 to go over things (welcoming doc, permanent weekly 1x1 timeslot) on Monday and another one on Friday to ensure week #1 went smoothly and learn about the painpoints.
+
+## Accounts setup
+Here is a list of accounts and access needed to be successful in Release Engineering.
+
+* SSO - we rely on [auth0](https://auth0.com/) across Mozilla for authentication and [LDAP](https://mana.mozilla.org/wiki/display/SYSADMIN/LDAP+Architecture) for authorization.
+Once given LDAP and you have created a permanent password, you can use that to login to the [portal](https://sso.mozilla.com/). From SSO, you should have links to various services from email, calendar, slack, mana, etc.
+* https://login.mozilla.com/ is where you can change a number of authentication/authorization access bits that you have control over. Each todo in this section assumes you have access to this page.
+* SSH - upload your public ssh key. It is a good idea to generate a separate ssh keypair from your personal one or any other that you have created in the past and use that explicitly for Releng and upload that. Follow this [SSH guidelines](https://wiki.mozilla.org/Security/Guidelines/OpenSSH#OpenSSH_client) doc on how to generate, configure, and use your ssh key.
+* PGP - We use pgp keys to share private information, secrets, and verify that the source came from someone we trust. Generate a keypair for this and upload your public key to https://keys.openpgp.org/ so others can find it. It would be really good if you could have other people sign your key, adding more trust that this key really belongs to you. You can use the the [pgp quickstart](https://mana.mozilla.org/wiki/display/SD/Generating+a+GPG+Public+Key) guide on mana or you can use the The [GNU Privacy Handbook](https://www.gnupg.org/gph/en/manual.html) for reference.
```

Comment:
```suggestion
* GPG - We use gpg keys to share private information, secrets, and verify that the source came from someone we trust. Generate a keypair and upload your public key to [OpenPGP](https://keys.openpgp.org/) so others can find it. It would be really good if you could have other people sign your key, adding more trust that this key really belongs to you. You can use the the [GPG quickstart](https://mana.mozilla.org/wiki/display/SD/Generating+a+GPG+Public+Key) guide on mana or you can use the The [GNU Privacy Handbook](https://www.gnupg.org/gph/en/manual.html) for reference.
```

GPG, PGP... so confusing... PGP is owned by Symantec (commercial license), OpenPGP is an email-centric service (but also used for identity trust services), and GPG (aka GnuPG) is the system/key format we use for signing commits/messages/data. I don't know why they made the names so similar. AFAIK GnuPG was based on PGP. Anyway... \<tableflip\>